conf/adminroles.conf.example

```diff
@@ -11,3 +11,7 @@ allowed_access_levels=User Manager
 [Security Event Manager]
 description=Security Events managements
 actions=SECURITY_EVENTS_READ,SECURITY_EVENTS_CREATE,SECURITY_EVENTS_UPDATE,SECURITY_EVENTS_DELETE,USERS_READ,NODES_READ
+
+[External API Receiver]
+description=Receives and execute external api calls
+actions=API_CALL_READ, API_CALL_CREATE, API_CALL_UPDATE, API_CALL_DELETE
```

Comment:
These nouns should be more descriptive, include "NTLM" in the naming.

Reply:
changed role to "Windows Event Receiver NTLM",
permission to "Windows Events READ" but accepts a POST requests - from PacketFence side, it's reading the windows events, not creating.